### ISSUE TITLE
Input and predicted car scaling is off

### ISSUE THREAD

**Issue by darrenjkt** [yuxumin/PoinTr#28]
Hi,

I ran PoinTr for the KITTI dataset as provided and visualized the input.npy and pred.npy with open3d. I'm aware that the car's pose has been normalized and therefore it seems odd to me that the scaling of the input.npy and pred.npy point clouds appear different? Does anyone else have this problem?

This was the command I ran. I did a fresh pull from the repo with no code modifications. 
```
bash ./scripts/test.sh 0     
--ckpts ./pretrained/PoinTr_KITTI.pth     
--config ./cfgs/KITTI_models/PoinTr.yaml     
--exp_name
```

Purple is the input.npy and grey is the pred.npy. Predicted point cloud looks smaller than the input point cloud.

![Screenshot from 2022-01-24 15-50-28](https://user-images.githubusercontent.com/39115809/150723876-04ce609f-0989-40c2-9113-f4690be10ebc.png)

**Comment by yuxumin** [comment]
Hi Darren,

Thanks for pointing it out. I check the KITTI models and find the same issue. It seems that PoinTr pretrained on PCN and then finetuned on the PCNCars can not adapt to the data on KITTI well.  Since the data on KITTI is normalized based on the bbox size,  there exist some gaps between the data on the KITTI and PCN (scale and inherent resolution). However, PoinTr and other completion models haven't be finetuning on the data from KITTI, so the robustness on scale of object is required.

![pic1](https://user-images.githubusercontent.com/29698233/150731410-a19d6612-230d-48a9-9d37-3c99606937dc.png)
![pic2](https://user-images.githubusercontent.com/29698233/150731571-66e04a78-b018-4e8f-a51e-b293af7984bd.png)

I visualize two samples from PCNCars and KITTI, blue for PCN, red for KITTI.

During the finetining on PCNCars, we only use *random dropping* to help the model adapt to the various input resolution. However, it seems that the scales of the inputs should be also taken into account.

Best!

**Comment by darrenjkt** [comment]
Thank you for double-checking this for me! 

I've been taking a look at various shape completion methods for use in my project and I realised that a lot of deep-learning-based methods require a normalized pose of the object. Is PoinTr and these other deep-learning-based methods a different sub-field of shape completion to traditional algorithms like "Ball Pivoting Algorithm" (Bernardini, 1999) which triangulates the surface using the original pose of the object?

**Comment by yuxumin** [comment]
Yeah, we didn't consider the pose of objects in completion. 
Deep-learning-based methods require that the training set matches the application scenario.  I think rotating the objects into a random pose during the training of deep-learning-based methods, the completion models can handle the completion tasks for objects with random poses.
